### PR TITLE
[keras/layers/convolutional/conv3d.py] Standardise docstring usage of "Default to"

### PR DESCRIPTION
**Pull request by SamuelMarks**
This is one of many PRs. Discussion + request to split into multiple PRs @ #17748

### CODE REVIEW

**Review by qlzh727 — CHANGES_REQUESTED** [review]
Please fix the lint error.

**Comment by SamuelMarks** [comment]
> Please fix the lint error.

Fixed

File: keras/layers/convolutional/conv3d.py [hunk 1]
```diff
@@ -83,11 +83,11 @@ class Conv3D(Conv):
...
-        (channels, spatial_dim1, spatial_dim2, spatial_dim3)`. It defaults to
-        the `image_data_format` value found in your Keras config file at
-        `~/.keras/keras.json`. If you never set it, then it will be
-        "channels_last". Note that the `channels_first` format is currently not
-        supported by TensorFlow on CPU.
+        (channels, spatial_dim1, spatial_dim2, spatial_dim3)`. When unspecified,
+        uses `image_data_format` value found in your Keras config file at
+        `~/.keras/keras.json` (if exists) else 'channels_last'. Note that the
+        `channels_first` format is currently not supported by TensorFlow on CPU.
+        Defaults to 'channels_first'.
```
Comment (qlzh727) [inline keras/layers/convolutional/conv3d.py:90]: I think the default is `channels_last`.

**Review by qlzh727 — CHANGES_REQUESTED** [review]

File: keras/layers/convolutional/conv3d.py (hunk 1 above)
Comment (SamuelMarks) [reply keras/layers/convolutional/conv3d.py]: Fixed

**Review by qlzh727 — APPROVED** [review]

**Pull request merged**